src/pages/account.tsx

```diff
@@ -1,12 +1,39 @@
 import React from 'react';
 import {
   AmplifyAuthenticator,
-  AmplifySignUp,
   AmplifySignOut,
+  AmplifySignUp,
 } from '@aws-amplify/ui-react';
+import {
+  AuthState,
+  onAuthUIStateChange,
+  CognitoUserInterface,
+} from '@aws-amplify/ui-components';
+import { useDispatch } from 'react-redux';
+import accountSlice from '../ducks/account/slice';
+import { useAccountState } from '../ducks/account/selectors';
+
+const AuthStateApp: React.FunctionComponent = () => {
```

Comment:
変更ありがとう！ごめん、ここも変更しないとBuildエラーになっちゃうね・・・:cat2:

```suggestion
const AccountPage: React.FunctionComponent = () => {
```

Reply:
気づかなかった…！修正ありがとう！🐱